### PR TITLE
pin flatbuffers to 0.8.4

### PR DESCRIPTION
**Pull request by ritchie46**
Fix for #238

### CODE REVIEW

File: arrow/Cargo.toml [hunk 1]
```diff
@@ -47,7 +47,7 @@ regex = "1.3"
 lazy_static = "1.4"
 packed_simd = { version = "0.3.4", optional = true, package = "packed_simd_2" }
 chrono = "0.4"
-flatbuffers = "^0.8"
+flatbuffers = "0.8.4"
```
Comment (returnString) [inline arrow/Cargo.toml:50]: I think this needs to be `< 0.8.5` or similar, as bare versions with no operator get interpreted as [caret requirements](https://doc.rust-lang.org/cargo/reference/specifying-dependencies.html#specifying-dependencies-from-cratesio).

**Review by jorgecarleitao — APPROVED** [review]
Thanks a lot, @ritchie46 , for your fast response and fix 👍

Let's hope that this is temporary and flatbuffers addresses this on their side of things.

File: arrow/Cargo.toml (hunk 1 above)
Comment (andygrove) [reply arrow/Cargo.toml]: You can do `"=0.8.4"`

Comment (andygrove) [reply arrow/Cargo.toml]: I know this from experience when they introduced const generics in 0.8.3 and that broke CI until we upgraded our Rust versions

Comment (ritchie46) [reply arrow/Cargo.toml]: > You can do `"=0.8.4"`

I go for that one,  that seems most explicit.

Comment (returnString) [reply arrow/Cargo.toml]: Yeah Andy's right, that's a better solution here - forgot that Cargo handles conflicting dep requirements relatively gracefully so doesn't matter if an Arrow user needs another `flatbuffers` version in their project. Been doing too much Python lately.

**Review by alamb — APPROVED** [review]

**Review by Dandandan — APPROVED** [review]

**Comment by alamb** [comment]
merging to get CI green

**Comment by ritchie46** [comment]
Thanks for the quick merge on this one!

**Pull request merged**